test/common/upstream/original_dst_cluster_test.cc

```diff
@@ -94,11 +94,11 @@ class OriginalDstClusterTest : public Event::TestUsingSimulatedTime, public test
 
   Stats::TestUtil::TestStore stats_store_;
   Ssl::MockContextManager ssl_context_manager_;
+  NiceMock<Event::MockDispatcher> dispatcher_;
```

Comment:
is this change needed?

Reply:
Same as https://github.com/envoyproxy/envoy/pull/14954#discussion_r574229756